### PR TITLE
[13.0][MIG] website_odoo_debranding

### PR DESCRIPTION
**Pull request by ougc27**
Hello everybody

This pull request has the migration of module website_odoo_debranding, i already test this and all works fine

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis status

**Comment by ougc27** [comment]
> Please check Travis status

Done!

File: website_odoo_debranding/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module remove Odoo branding on website:
+
+* Remove "Create a free website with Odoo" from footer
```
Comment (astirpe) [inline website_odoo_debranding/readme/DESCRIPTION.rst]: Should be:

```suggestion
* Remove "Powered by Odoo - Create a free website" from footer
```

File: website_odoo_debranding/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#. Go to your home page.
+#. Enable or disable *Customize > Footer Copyright > Remove Odoo Promotional
```
Comment (astirpe) [inline website_odoo_debranding/readme/USAGE.rst]: I don't find the "Remove Odoo Promotional" link. Could you check?

**Review by astirpe — CHANGES_REQUESTED** [review]

File: website_odoo_debranding/readme/DESCRIPTION.rst (hunk 1 above)
Comment (ougc27) [reply website_odoo_debranding/readme/DESCRIPTION.rst]: Done! :+1:

File: website_odoo_debranding/readme/USAGE.rst (hunk 2 above)
Comment (ougc27) [reply website_odoo_debranding/readme/USAGE.rst]: Yes you are right, this is not necessary, Done! :+1:

**Review by astirpe — APPROVED** [review]

**Comment by ougc27** [comment]
Hello @astirpe could you help me to ask another review, please? Thank you for your review

**Review by feketemihai — APPROVED** [review]

File: website_odoo_debranding/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2016 Tecnativa, S.L. - Vicent Cubells
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Remove Odoo Branding from Website",
+    "version": "13.0.1.0.0",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline website_odoo_debranding/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Review by MiquelRForgeFlow — APPROVED** [review]

File: website_odoo_debranding/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1 @@
+#.No additional configuration is needed. The legend "Powered by Odoo" from footer is removed automatically.
```
Comment (Tardo) [inline website_odoo_debranding/readme/USAGE.rst:1]:
```suggestion
#. No additional configuration is needed. The legend "Powered by Odoo" from footer is removed automatically.
```

**Review by Tardo — CHANGES_REQUESTED** [review]

**Comment by Tardo** [comment]
Any progress here?

**Comment by ougc27** [comment]
Hello @Tardo 

Sorry I could not retake this PR, but the suggested changes are done :+1:

**Review by Tardo — APPROVED** [review]

**Comment by Tardo** [comment]
Nice! run pre-commit ;)

**Comment by Tardo** [comment]
Well, i go to supersede this PR and apply the pre-commit changes.

**Comment by pedrobaeza** [comment]
Superseded by #708

**Pull request closed without merging**